### PR TITLE
Addressing rubocop warnings

### PR DESCRIPTION
**Pull request by khandpur**
As in here: https://travis-ci.org/google/EarlGrey/jobs/263713604

### CODE REVIEW

File: gem/lib/earlgrey/configure_earlgrey.rb [hunk 1]
```diff
@@ -379,8 +379,8 @@ def copy_swift_files(project, target, swift_version = nil)
       src_swift = File.join(src_root, "Swift-#{swift_version}", src_swift_name)
 
       unless File.exist? src_swift
-        puts_magenta "EarlGrey.swift for version #{swift_version} not found. " +
-            "Falling back to version 3.0."
+        puts_magenta "EarlGrey.swift for version #{swift_version} not found. " \
```
Comment (tirodkar) [inline gem/lib/earlgrey/configure_earlgrey.rb]: This is double quotes while the other is single.

Comment (khandpur) [inline gem/lib/earlgrey/configure_earlgrey.rb]: That was one of the warnings on here:
https://travis-ci.org/google/EarlGrey/jobs/263713604

**Comment by khandpur** [comment]
And it passes with this PR:
https://travis-ci.org/google/EarlGrey/jobs/263718922

**Comment by wuhao5** [comment]
Thanks!

**Pull request merged**